packages/circuits/circuits/extractor.circom

```diff
@@ -80,8 +74,4 @@ template Extractor(max_num_bytes) {
     for (var i = 0; i < 4; i++) {
         four_digit[i] <== data[i + 2] * selector[1];
     }
-
```

Comment:
@vuvoth This is being done in QR_Verify circuit.

Reply:
I think this should be done in extractor, because this circuit care about extract information.